haskell-build/orb.yml

```diff
@@ -68,6 +68,7 @@ commands:
             - run:
                 name: Searching for Cabal file to build
                 command: |
+                  if [ -f cabal.project ]; do -----
                   BUILD_CABAL_FILE=$((ls ${CIRCLE_PROJECT_REPONAME}.cabal || ls *.cabal) | head -n 1)
```

Comment:
This line fails if there's no top-level cabal file. I'm just trying to work out how to fix it.